packages/checkout/panel/style.scss

```diff
@@ -6,6 +6,13 @@
 	}
 }
 
+.wc-block-components-panel.has-border.no-top-border {
+	@include with-translucent-border( 1px 0 );
+	&::after {
+		border-top-width: 0;
+	}
+}
+
```

Comment:
You should be able to accomplish the same with the mixin:

```suggestion
.wc-block-components-panel.has-border.no-top-border {
	@include with-translucent-border( 0 0 1px );
}
```

I also think the selector could be renamed from `.wc-block-components-panel.has-border.no-top-border` to `.wc-block-components-panel.has-bottom-border`.

And, ideally this would be exported from a `TotalsWrapper` component so 3rd-party extensions don't need to call our CSS class names. Sorry if I'm too annoying with that! :smile: It's not blocking this PR, though.

Reply:
Yep, the `TotalsWrapper` change is on my to-do list! I think for now I can completely remove `no-top-border` and work on the `TotalsWrapper` changes. In the meantime there will be a double border.

![image](https://user-images.githubusercontent.com/5656702/120628356-8148cf80-c45c-11eb-9017-6827a48e1b55.png)